src/controller/python/test_client/README.md

```diff
@@ -0,0 +1,13 @@
+# python-xmlrpc-docker
+Sample client program using XML-RPC to do RPC across multiple containers running chip stack.
+
+
+Setup:
+1. Build the connectedhomeip repo or the certification fork repo by running the following scripts in the scripts directory:
+    To just build the container:`source scripts/build_container.sh`
+    To build both the sdk and the container: `source scripts/build_sdk_container.sh`
```

Comment:
If you add the executable bit to these scripts you could just have `./scripts/build_...sh`